lib/integer.gd

```diff
@@ -990,6 +990,7 @@ DeclareGlobalFunction( "SmallestRootInt" );
 ##  <Description>
 ##  prints the prime factorization of the integer <A>n</A> in human-readable
 ##  form.
+##  See also <Ref Func="StringPP"/>.
```

Comment:
It *is* documented. I only added links between the documentation entries. And it's been around for ages. And there are more things of this kind. For a time, I think people were to eager to add lots of "convenient" special purpose functions like these, which end up being hard to discover, and thus duplicated (and even then they are not used much, because everybody has a hard time discovering them).